### PR TITLE
SUS-1567 | introduce StaticUserPages extension

### PR DESCRIPTION
**Pull request by macbre**
https://wikia-inc.atlassian.net/browse/SUS-1587

Introduce an extension that will be used site-wide to render user pages for bot accounts using content stored in i18n messages.

Covered accounts: FANDOM(bot), Fandom(Bot) and Wikia(Bot).

Code in action:

* http://sandbox-s2.poznan.wikia.com/wiki/U%C5%BCytkownik:FANDOM
* http://sandbox-s2.poznan.wikia.com/wiki/U%C5%BCytkownik:FANDOMbot

Code-generated pages respond with `HTTP 200` and are cached on CDN layer for 24h.

### CODE REVIEW

File: extensions/wikia/StaticUserPages/StaticUserPagesHooks.class.php [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+	}
+
+	/**
+	 * Return i18n message name for a given title (or false if there's none)
+	 *
+	 * @param Title $title
+	 * @return string|false
+	 */
+	static public function getMessageForTitle( Title $title ) {
+		return self::USER_PAGES_MESSAGES[ $title->getText() ] ?: false;
```
Comment (WikiaTech) [inline extensions/wikia/StaticUserPages/StaticUserPagesHooks.class.php]: MINOR Remove the literal "false" boolean value. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1125)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue
* ![MINOR][MINOR] 1 minor

Watch the comments in this conversation to review them.

[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

File: extensions/wikia/StaticUserPages/StaticUserPagesHooks.class.php (hunk 1 above)
Comment (WikiaTech) [inline extensions/wikia/StaticUserPages/StaticUserPagesHooks.class.php]: MINOR Remove the literal "false" boolean value. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1125)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue
* ![MINOR][MINOR] 1 minor

Watch the comments in this conversation to review them.

[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Review by mszabo-wikia — APPROVED** [review]

File: extensions/wikia/StaticUserPages/StaticUserPagesHooks.class.php (hunk 1 above)
Comment (WikiaTech) [inline extensions/wikia/StaticUserPages/StaticUserPagesHooks.class.php:39]: MINOR Remove the literal "false" boolean value. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1125)

**Pull request merged**